ideascube/tests/test_settings.py

```diff
@@ -14,6 +14,12 @@ def setting_module(request):
 
 
 def test_setting_file(setting_module):
+    from ideascube.forms import UserImportForm
+
     settings = importlib.import_module(setting_module, package="ideascube")
 
     assert isinstance(getattr(settings, 'IDEASCUBE_NAME', ''), str)
+
+    for name, label in getattr(settings, 'USER_IMPORT_FORMATS', []):
```

Comment:
`label` is not used.
(It is surprising that pyflake doesn't complain about this)